### PR TITLE
WSTEAMA-319: Implement visually hidden H1 for the tipo homepage

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves [JIRA WSTEAMA-319
](https://jira.dev.bbc.co.uk/browse/WSTEAMA-319)

Overall changes
======

- Adds newly created`VisuallyHiddenText` component to Home Page 

Code changes
======
- Uses new`VisuallyHiddenText` component for the H1 on the HomePage (created to latest component standards)
- Adds unit tests to prove that `h1` prop is passed to generate an h1, render offscreen text and to validate that an `h1` is returned to the DOM

Testing
======
- [x] Manual testing locally & inspecting HTML to ensure H1 present
- [x] Unit test
- [ ] A11y testing
  - [ ] axe browser extension
  - [ ] View Document outline via Web Developer Tools browser extension
  - [ ] Voiceover on MacOS 
  - [ ] Screenreader on android using [storybook permalink](https://wsteama-319-visually-hidden-h1-tipoho--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&knob-Select%20a%20service=kyrgyz&id=pages-home-page--example&viewMode=story)
  - [ ] Screenreader on iPhone using [storybook permalink](https://wsteama-319-visually-hidden-h1-tipoho--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&knob-Select%20a%20service=kyrgyz&id=pages-home-page--example&viewMode=story)

Helpful Links
======

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
 import {
-  CurationData,
   VisualProminence,
   VisualStyle,
-} from '#app/models/types/curationData';
-import Curation from '#app/components/Curation';
+  CurationData,
+} from '../../models/types/curationData';
+import Curation from '../../components/Curation';
+import VisuallyHiddenText from '../../legacy/psammead/psammead-visually-hidden-text/src';
```
Comment (amoore108) [inline src/app/pages/HomePage/HomePage.tsx]: To counter the changes about the components standards in the other PR 😅 , this component actually looks pretty small and could be a quick win to bring it in to main `components` folder.

Its likely we will be using it for the likes of Live pages as well. Doesn't need to be part of this, but may be a good one to convert over?

Comment (karinathomasbbc) [reply src/app/pages/HomePage/HomePage.tsx]: We have allowed a little bit of extra time this sprint to do this. Let me do it in a separate PR, and use it here. Thanks for the suggestion.

**Review by amoore108 — APPROVED** [review]

**Review by mfonofm — APPROVED** [review]

File: src/app/pages/HomePage/HomePage.tsx (hunk 1 above)
Comment (karinathomasbbc) [reply src/app/pages/HomePage/HomePage.tsx]: This has now been completed as part of #10745

File: src/app/components/VisuallyHiddenText/index.test.tsx [hunk 2]
```diff
@@ -9,4 +9,10 @@ describe('VisuallyHiddenText', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+  it('should render off screen text for screen readers as an h1', () => {
```
Comment (karinathomasbbc) [inline src/app/components/VisuallyHiddenText/index.test.tsx:13]: *teeny tiny nitpick*: whitespace
```suggestion
  
  it('should render off screen text for screen readers as an h1', () => {
```

File: src/app/components/VisuallyHiddenText/index.tsx [hunk 3]
```diff
@@ -3,8 +3,24 @@ import { jsx } from '@emotion/react';
...
+}
+
+const VisuallyHiddenText = ({
+  children,
+  id,
+  tabIndex,
+  as,
+}: PropsWithChildren<VisuallyHiddenTextProps>) => {
+  const Component = as || 'span';
+  return (
```
Comment (karinathomasbbc) [inline src/app/components/VisuallyHiddenText/index.tsx:19]: *nitpick*: new line
```suggestion

  const Component = as || 'span';
  return (
```

**Review by karinathomasbbc — COMMENTED** [review]
Awesome, we managed to avoid using the `Text` component, and hopefully this will work for all of the other components going forward 🤞 . Thanks for sorting this!

**Comment by pvaliani** [comment]
Use of `Text` was polluting our snapshots and creating a more opinionated set of snapshots and `VisuallyHiddenText` component.

By passing `as` or `span` to `Component` we more closely follow the legacy `VisuallyHiddenText` component with all the type-checking benefits of our new one.

**Review by pvaliani — APPROVED** [review]

**Review by amoore108 — APPROVED** [review]

**Review by mfonofm — APPROVED** [review]
looks good 👍

**Pull request merged**